### ISSUE TITLE
How to Solve Sub-Menu Accessibility Issues in Astra Theme

### ISSUE THREAD

**Issue by silversufer** [brainstormforce/astra#5040]
I am using the free Astra theme and have a question regarding the menu and sub-menu. When I hover over the parent menu item with my mouse, the sub-menu is automatically expanded and displayed. Similarly, when I use the Tab key to navigate, the sub-menu is also automatically expanded, causing issues with screen readers. Ideally, the sub-menu should not open automatically when using the Tab key on the parent element unless the Tab key is on the button element, where pressing Enter will open the sub-menu.

I noticed in the theme customizer that it is possible to set and configure the animation for the sub-menu, but unfortunately, it does not have the desired effect for me. As a result, I am unable to configure the menu navigation to be WCAG compliant for accessibility, even though the HTML structure looks good.

When I use the Tab key on the parent element, I automatically navigate through the sub-menu items. However, I do not have a way to expand or collapse the sub-menu items using the button, and clicking on another link is required to close the sub-menu. This is not an ideal user experience.

The desired behavior for the navigation menu is as follows:

-   Use the Tab key to navigate along the main menu (Parent).
-   Press the Tab key on the button element to open the sub-menu.
-   Use the up and down arrow keys to navigate within the sub-menu.
-   Press the Tab key again to either continue navigating in the main menu (Parent) or switch to the sub-submenu (Level 3), if available.

I am using the latest version of the theme and the latest version of Wordpress.

**Comment by neerajmasai** [comment]
Thanks for writing in @silversufer 
I did a quick test with Astra 4.1.6 according to that when using tab navigation over the Parent Menu Element the Sub Menu does not open automatically but requires the user to press the "Enter" key in order to open the sub menu.
Please refer to the Screenshot: https://d.pr/i/mdpd1S

**Comment by neerajmasai** [comment]
Hey @silversufer,
As the issue is not reproducible and we have not received any further inputs from your side, we are closing this issue for now. If the issue persists please feel free to reach out to us.
Warm Regards,
Neeraj
